service/src/test/java/org/sunbird/learner/actors/LearnerStateActorTest.java

```diff
@@ -196,4 +202,119 @@ public void testForGetContentByContentIds() {
     Response res = probe.expectMsgClass(duration("10 second"), Response.class);
     Assert.assertNotNull(res);
   }
+
+  @Test
+  public void testGetCourseByUserIdAndCourseMetadata() {
```

Comment:
[testName]Success

What is metadata here ? impove the test-name - if probably testGetCourseByUserIdAndCourseBatchesSuccess - is what funciton is doing I guess